packages/phrases/src/types.ts

```diff
@@ -10,6 +10,7 @@ export type ResourceKey = string | Record<string, unknown>;
 const languageCodeAndDisplayNameMappings: Record<LanguageKey, string> = {
   en: 'English',
   fr: 'Français',
+  'pt-PT': 'Portuguese',
```

Comment:
We prefer using native language phrase as the value. So I guess for here it should be:
```ts
'pt-PT': 'Português',
```

Reply:
No worries! 👌🏻